doc/authentication-backends/LDAP-authentication-module.md

```diff
@@ -0,0 +1,53 @@
+## Overview
+
+An LDAP authentication module.
+It provides read-only abstraction over LDAP directory.
+User credentials are verified by performing LDAP bind with username and password provided by a client.
+
+Requires SASL PLAIN method.
+
+## Configuration options
+
+* **ldap_base:**
+    * **Description:**  LDAP base directory which stores users accounts.
+    * **Values:** String
+    * **Default:** This option is required
+
+* **ldap_uids:**
+    * **Description:**  LDAP attribute which holds a list of attributes to use as alternatives for getting the JID.
```

Comment:
an LDAP attribute holding a list of attributes to use as alternatives for getting the JID.